### DISCUSSION TITLE
Componente PoLookup com a  propriedade p-multiple não mantém os itens selecionados ao recarregar a página

### DISCUSSION THREAD

**Discussion by mkazimoto** [po-ui/po-angular#1247] [Assuntos gerais]
Bom dia!

Identificamos um bug no componente PoLookup com a  propriedade p-multiple="true" que não mantém os itens selecionados ao recarregar a página bindado com uma lista do contexto.

Este problema não acontece com o componente PoMultiSelect que mantém os itens selecionados ao recarregar a página.
![Simulação do bug](https://user-images.githubusercontent.com/5958365/161764620-beca7ba6-c482-4023-8bab-f7b2a88d98ac.png)

**Comment by jnrpalma** [comment]
Boa tarde @mkazimoto ! 
Você poderia realizar uma simulação no stackblitz (https://stackblitz.com/edit/po-ui) para melhor entendermos ? Obrigado.
